content/blogs/2023-09-18-kestra-terraform-partnership.md

```diff
@@ -0,0 +1,57 @@
+## Elevating Your Data Workflow With Kestra's Terraform Provider ##
+
+![Terraform Kestra](/blogs/2023-09-18-kestra-terraform-partnership/terraform-kestra.png)
+
+Our Terraform provider is a significant step in aligning data orchestration with infrastructure-as-code best practices. With this integration, configuring Kestra workflows and tasks directly within your Terraform scripts is a seamless experience. It's an integrated approach to streamline and automate all your flows.
+
+
+### Breaking the Production Barrier: The Kestra and Terraform Solution ###
+
+Moving data pipelines from development to production comes with its own set of challenges. Be it environment setups, database schema access, or cloud resource permissions, the list seems endless. The alliance between Kestra and Terraform aims to bridge this gap by integrating open-source tools that make this transition easier for developers. Learn more about [Everything as Code for Data Infrastructure solution](https://levelup.gitconnected.com/programmable-data-infrastructure-is-finally-within-reach-83fb8e6392ac).
+
+
+### Supercharge Your CI/CD with Terraform & Kestra ###
+Bring Kestra into your CI/CD pipelines using tools like GitHub Actions. By doing so you can automate your infrastructure provisioning but also deploy your Kestra flows in the CI/CD model. With a GitHub Actions workflow, you can also validate and deploy flows directly into different namespaces ensuring that your data workflows are as agile and adaptable as your infrastructure.
+<br>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?si=jXSj49xwStR5Wqk8&amp;list=PLEK3H8YwZn1olYeQ7qbGdY5XQySKCNLUr" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+<br>
+
+### Unlocking Data Efficiency: 5 Million Executions and 4,000+ Flows with Kestra and Terraform ###
+
+**Leroy Merlin France** achieved new heights in data agility by integrating Kestra and Terraform, both employing declarative language and fully embracing Git-based version control. They used Terraform for automated deployments of BigQuery assets like datasets, tables, views, and ACLs, all part of a streamlined CI/CD process. Upon adopting Kestra, a custom Terraform provider was developed to bring the same level of version control and automation to Kestra’s resources. This harmonized architecture has led to over **5 million pipeline executions** each month, and more than **4.000+ data flows created**, and a staggering 900% increase in data activity, all while fostering team-wide confidence during production deployments. [Learn more about this amazing use case](https://kestra.io/blogs/2023-08-16-datamesh)
+
+--- 
+
+## What’s Next ##
+
+Our collaboration is just the beginning. Your insights and contributions can greatly influence the utility and success of this integration. Stay tuned for more updates and opportunities.
```

Comment:
```suggestion
Our collaboration is just the beginning. Your insights and contributions can greatly influence the utility and success of this integration. Share your feedback via [Kestra Community Slack](https://kestra.io/slack) or open [a GitHub issue](https://github.com/kestra-io/kestra).
```

it's weird to mention that user's insights/feedback is welcome without showing how they can provide it

Reply:
Yep right! just dumb copy pasted the final message with the link